src/commands/generateEditorConfig.ts

```diff
@@ -0,0 +1,45 @@
+import {exists, writeFile} from 'fs';
+import {workspace, window} from 'vscode';
+import {Utils} from '../utils';
+
+const supportedProperties = ['indent_style', 'indent_size', 'tab_width'];
```

Comment:
We also support `insert_final_newline` and `trim_trailing_whitespace`, so why are they not added to this list?